src/background_remover.cc

```diff
@@ -184,7 +184,7 @@ cv::Mat BackgroundRemover::getMaskFromOutput() {
 
         ret.forEach([&](unsigned char &is_background, const int loc[]) -> void {
             const auto p = cv::Point(loc[1], loc[0]);
-            is_background = maxIdx(labels(p)) != person_label;
+            is_background = 255 * (maxIdx(labels(p)) != person_label);
```

Comment:
is_background isn’t really appropriate now anymore. background_depth or so? (Also, please make this `maxIdx(..) == bar ? 0 : 255` or something more readable than a multiplication with a bool.)